### DISCUSSION TITLE
What are the steps necessary to start with a language other then the default english

### DISCUSSION THREAD

**Discussion by undersound** [area17/twill#2194] [Q&A]
Let's say I want to create a site with german content (later on I want to add the english language)
What are the steps necessary to start with a default content language of German (de) For clarity I am not talking about the interface language but the content language.

Is it only a mattter of changing the locale in _config/translatable.php_ from **en** to **de**?

**Comment by ifox** [comment]
Hi @undersound, that's correct.

**Reply by undersound** [reply]
Thanks @ifox . 

What are the consequences when you have started with a "wrong" default language and there is already content created? To stay with the example above....I started with the default language EN and now I want change it to DE. 

Is it only matter of changing the langcodes for the content or is there more to it?

**Reply by ifox** [reply]
In the translations and slugs table you will need to update the locale column content accordingly, yes. If you're using blocks, the JSON in db contains locale keys too.
